Add contentcreated date-time field

NiallBunting opened the pull request:
This will capture the date-time when the original asset was created
i.e. The time a photographer takes the photo according to the camera.

Added the following to the 1.2 schema:
```
+    "contentcreated": {
+      "description": "The date and time when the asset was created",
+      "type": "string",
+      "format": "date-time"
+    },
```

Implements: IAI-1316

Commit:
Add contentcreated date-time field

This will capture the date-time when the original asset was created
i.e. The time a photographer takes the photo according to the camera.

Added the following to the 1.2 schema:
+    "contentcreated": {
+      "description": "The date and time when the asset was created",
+      "type": "string",
+      "format": "date-time"
+    },

Implements: IAI-1316

iyoung reviewed (approved):
Looks good to me, one non-required new date-time field contentcreated added

petersow commented on content/ninjs/pa-ninjs-schema_1.3.json:
```diff
@@ -0,0 +1,425 @@
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://schema.pressassociation.io/content/ninjs/pa-ninjs-schema_1.3.json#",
+  "type": "object",
+  "title": "PA IPTC ninjs - News in JSON - version 1.1 (Derived from IPTC - News in JSON - version 1.1)",
```
version 1.1 -> version 1.3 (I forgot to update for 1.2)

petersow commented on content/ninjs/pa-ninjs-schema_1.3.json:
```diff
@@ -0,0 +1,425 @@
+      "additionalProperties": false,
+      "patternProperties": {
+        "^[a-zA-Z0-9]+": {
+          "$ref": "http://schema.pressassociation.io/content/ninjs/pa-ninjs-schema_1.1.json#"
```
bump to 1.3, to self refer

NiallBunting committed:
Bump versions to the latest

Missed some versions when creating the file. Bump them 1.1 -> 1.3.

Relates-to: IAI-1316

petersow commented:
Could we add a 1.3 version of the list please?

Commit:
Add PA list 1.3 to point to ninjs-schema 1.3

This adds a list schema that references the 1.3 version of the item.

Relates-to: IAI-1316
